### PR TITLE
Create advanced cluster with YAML

### PR DESCRIPTION
**Pull request by vacas5**
@cmastr here's a first draft but it should probably be taken over by someone

## Jira Ticket

Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

Closes D2IQ-81810

## Description of changes being made

Document the workflow for users to use the new YAML editor in the UI.

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [x] Create new PRs against `develop`, or `main` for hotfixes to production.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](./CONTRIBUTING.md) for more information.

### CODE REVIEW

**Comment by cmastr** [comment]
Asked @mwrightd2 to take a look at these docs.

**Comment by cmastr** [comment]
These docs need to go into 2.2 as well please.

File: pages/dkp/kommander/2.1/clusters/creating-konvoy-cluster-aws/index.md [hunk 1]
```diff
@@ -18,11 +18,11 @@ From the **Add Cluster** menu, select the **Create Konvoy Cluster** and provide
...
+Select the pre-configured [AWS infrastructure provider](../../operations/infrastructure-providers/configure-aws-infrastructure-provider-static-credentials/) (or [AWS role credentials](../../operations/infrastructure-providers/configure-aws-infrastructure-provider-roles/)) to display the remaining options specific to AWS.
 
-- **Region**: Select the datacenter region to reveal selectable Availability Zones.
+- **Region**: Select the data center region to reveal selectable Availability Zones.
 - **Add Infrastructure Provider Tags**: Specify tags to be set on all resources created in your infrastructure provider for this cluster. You can add up to 10 tags in this form. Different infrastructure providers have varying restrictions on their tags. See the [AWS Tags User Guide][aws-tags] for more information on using tags in AWS.
 
 Click **Create** to begin provisioning the Konvoy cluster. This step may take a few minutes, and the cluster may fail to join initially if it tries to join before the cluster is fully ready. It will retry to join automatically and should resolve once the cluster is fully provisioned.
```
Comment (millerjem) [inline pages/dkp/kommander/2.1/clusters/creating-konvoy-cluster-aws/index.md:28]: I would reframe this in a more positive manner. Something like... This step or phase may take a few minutes to gain consistency based on cluster readiness and or completion/deployment of the cluster addons.

File: pages/dkp/kommander/2.1/clusters/creating-konvoy-cluster-aws/index.md [hunk 2]
```diff
@@ -18,11 +18,11 @@ From the **Add Cluster** menu, select the **Create Konvoy Cluster** and provide
 - **Workspace**: The workspace where this cluster belongs.
 - **Kubernetes Version**: The initial version of Kubernetes to install on the cluster.
 - **Name**: A valid Kubernetes name for the cluster.
-- **Add Labels**: By default, your cluster has labels that reflect the infrastructure provider provisioning. For example, your AWS cluster may be labelled with the datacenter region and `provider: aws`. Cluster labels are matched to the selectors created for [projects][projects]. Changing a cluster label may add or remove the cluster from projects.
+- **Add Labels**: By default, your cluster has labels that reflect the infrastructure provider provisioning. For example, your AWS cluster may have a label for the data center region and `provider: aws`. Cluster labels are matched to the selectors created for [projects][projects]. Changing a cluster label may add or remove the cluster from projects.
```
Comment (millerjem) [inline pages/dkp/kommander/2.1/clusters/creating-konvoy-cluster-aws/index.md:21]: We'll need documentation on how on the Kommander configmap non-standard regions such as us-gov-east-1, us-iso-east-1.c2s.ic.gov. As far as the labels, please provide examples that the end-user will need to use or modify?

Comment (vacas5) [reply pages/dkp/kommander/2.1/clusters/creating-konvoy-cluster-aws/index.md:21]: I don't believe any labels are required for the user to add, simply optional.

Good call about configuring the custom regions

**Review by cneth — APPROVED** [review]

**Comment by cmastr** [comment]
These docs need to be ported over to 2.2 documentation as well before we merge. 

https://github.com/mesosphere/dcos-docs-site/tree/develop/pages/dkp/konvoy/2.2

**Comment by vacas5** [comment]
@cmastr that will need to happen in another PR off the `develop` branch, actually

**Comment by cmastr** [comment]
Need to move this to the 2.1.1 branch?

File: .github/styles/write-good/Weasel.yml [hunk 3]
```diff
@@ -124,7 +124,6 @@ tokens:
   - normally
   - obediently
   - occasionally
-  - only
```
Comment (msdolbey) [inline .github/styles/write-good/Weasel.yml:127]: ❤️

File: pages/dkp/kommander/2.1/clusters/creating-konvoy-cluster-advanced/index.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: Advanced Creation of Konvoy Clusters
+title: Advanced Creation of Konvoy Clusters
+menuWeight: 35
+excerpt: A guide for advanced creation of Konvoy clusters
+---
+
+## Advanced Cluster Creation
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/clusters/creating-konvoy-cluster-advanced/index.md:10]:
```suggestion
## Advanced cluster creation
```

File: pages/dkp/kommander/2.1/clusters/creating-konvoy-cluster-advanced/index.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+<p class="message--warning"><strong>WARNING: </strong>
+This feature is for advanced users and users in unique environments only. We highly recommend using other documented methods to create clusters whenever possible.
+</p>
+
+### Generate cluster objects
+
+Depending on your infrastructure, Konvoy can generate a set of cluster objects that can be customized for unusual use cases. Visit the [documentation for AWS](https://docs.d2iq.com/dkp/konvoy/2.1/choose-infrastructure/aws/advanced/new/#create-a-new-aws-kubernetes-cluster) for an example on how to use the `--dry-run` and `--output` flags to create a set of cluster objects.
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/clusters/creating-konvoy-cluster-advanced/index.md:18]:
```suggestion
Depending on your infrastructure, Konvoy can generate a set of cluster objects that can be customized for unusual use cases. Visit the [documentation for AWS](/dkp/konvoy/2.1/choose-infrastructure/aws/advanced/new/#create-a-new-aws-kubernetes-cluster) for an example on how to use the `--dry-run` and `--output` flags to create a set of cluster objects.
```

File: pages/dkp/kommander/2.1/clusters/creating-konvoy-cluster-advanced/index.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: Advanced Creation of Konvoy Clusters
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/clusters/creating-konvoy-cluster-advanced/index.md:4]: We'll probably want to rename this to be "DKP clusters" in future versions.

File: pages/dkp/kommander/2.1/clusters/creating-konvoy-cluster-aws/index.md (hunk 1 above)
Comment (msdolbey) [reply pages/dkp/kommander/2.1/clusters/creating-konvoy-cluster-aws/index.md]: @millerjem - made some small edits - this look good now?

**Review by msdolbey — APPROVED** [review]
👍 
Looks good

**Comment by msdolbey** [comment]
Closing this one in favor of: https://github.com/mesosphere/dcos-docs-site/pull/3980 (for merging into a diff feature branch for the patch)

**Pull request closed without merging**